### PR TITLE
Added 'CRUD' tests to ContentView via API.

### PR DESCRIPTION
**Pull request by omaciel**
This PR does the following:

* Adds a couple of new 'CRUD' test cases for ContentView via the API interface;
* Renames existing test case names for more user friendly names;
* Removes extra blank lines from docstrings;

All tests have been run and passed.

### CODE REVIEW

**Comment by JacobCallahan** [comment]
ACK

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -96,7 +94,7 @@ def test_cv_clone_within_same_env(self):
         promote(cloned_cv.read().version[0], lc_env.id)
 
     @run_only_on('sat')
-    def test_cv_clone_within_diff_env(self):
+    def test_positive_clone_within_diff_env(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: s/within/with?

File: tests/foreman/api/test_contentview.py [hunk 2]
```diff
@@ -165,14 +160,13 @@ def test_cv_associate_puppet_repo_negative(self):
             ).create()
 
     @run_only_on('sat')
-    def test_cv_associate_composite_dupe_repos_negative(self):
+    def test_negative_associate_composite_dupe_content(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: this does not look like a composite test? may be remove `composite` from the name?

File: tests/foreman/api/test_contentview.py [hunk 3]
```diff
@@ -187,14 +181,13 @@ def test_cv_associate_composite_dupe_repos_negative(self):
         self.assertEqual(len(content_view.read().repository), 0)
 
     @run_only_on('sat')
-    def test_cv_associate_composite_dupe_modules_negative(self):
+    def test_negative_associate_composite_dupe_modules(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: this does not look like a composite test? may be remove composite from the name?

File: tests/foreman/api/test_contentview.py [hunk 4]
```diff
@@ -348,7 +355,7 @@ def test_positive_publish_2(self):
         for cvv in content_view.read().version:
             self.assertGreater(cvv.read_json()['package_count'], 0)
 
-    def test_publish_composite_cv_once_1(self):
+    def test_positive_publish_composite_single_content_once(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: test_positive_publish_once_composite_single_content_view?

File: tests/foreman/api/test_contentview.py [hunk 5]
```diff
@@ -366,7 +372,7 @@ def test_publish_composite_cv_once_1(self):
         composite_cv.publish()
         self.assertEqual(len(composite_cv.read().version), 1)
 
-    def test_publish_composite_cv_once_2(self):
+    def test_positive_publish_composite_multiple_content_once(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: test_positive_publish_once_composite_multiple_content_views?

File: tests/foreman/api/test_contentview.py [hunk 6]
```diff
@@ -385,7 +390,7 @@ def test_publish_composite_cv_once_2(self):
         composite_cv.publish()
         self.assertEqual(len(composite_cv.read().version), 1)
 
-    def test_publish_composite_cv_multiple_1(self):
+    def test_positive_publish_composite_single_content_multiple(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: test_positive_publish_multiple_composite_single_content_view?

File: tests/foreman/api/test_contentview.py [hunk 7]
```diff
@@ -406,7 +410,7 @@ def test_publish_composite_cv_multiple_1(self):
             composite_cv.publish()
             self.assertEqual(len(composite_cv.read().version), i + 1)
 
-    def test_publish_composite_cv_multiple_2(self):
+    def test_positive_publish_composite_multiple_content_multiple(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentview.py]: test_positive_publish_multiple_composite_multilple_content_views?

**Comment by sthirugn** [comment]
ACK pending minor comments

**Comment by omaciel** [comment]
@sthirugn comments have been addressed

**Comment by sthirugn** [comment]
ACK

**Comment by elyezer** [comment]
:+1:

**Pull request merged**